internal/pkg/manifest/testdata/backend-svc-customhealthcheck.yml

```diff
@@ -39,4 +39,6 @@ exec: true     # Enable running commands in your container.
 #  test:
 #    count: 2               # Number of tasks to run for the "test" environment.
 #    deployment:            # The deployment strategy for the "test" environment.
-#       rolling: 'recreate' # Stops existing tasks before new ones are started for faster deployments.
\ No newline at end of file
+#       rolling: 'recreate' # Stops existing tasks before new ones are started for faster deployments.
+storage:
+  readonly_fs: true
```

Comment:
Can we move the `storage` setting under  `exec: true`? Ideally the `#environments` comment is the last thing in their manifest file 🥺